browser/ui/BUILD.gn

```diff
@@ -466,10 +467,20 @@ source_set("ui") {
 
   if (brave_wallet_enabled && !is_android) {
     sources += [
+      "views/toolbar/wallet_button.cc",
+      "views/toolbar/wallet_button.h",
       "webui/brave_wallet_ui.cc",
       "webui/brave_wallet_ui.h",
+      "webui/wallet_panel/wallet_panel_page_handler.cc",
```

Comment:
can we put this in a separate directory so we can track deps better?

Reply:
Addressed here: 405f7426f6aa9f42682048bd173b31a52b9b8f62